Source/EThree.swift

```diff
@@ -73,33 +73,89 @@ import VirgilCrypto
 
     private var groupManager: GroupManager?
 
+#if os(iOS)
+    // swiftlint:disable line_length
+
     /// Initializer
     ///
     /// - Parameters:
     ///   - identity: User identity
     ///   - tokenCallback: callback to get Virgil access token
-    ///   - changedKeyDelegate: `ChangedKeyDelegate` to notify about changes of User's keys
+    ///   - biometricProtection: will use biometric or passcode protection of key if true. Default value - false.
+    ///   - biometricPromt: User promt for UI
+    ///   - changedKeyDelegate: [ChangedKeyDelegate](x-source-tag://ChangedKeyDelegate) to notify about changes of User's keys
     ///   - storageParams: `KeychainStorageParams` with specific parameters
     /// - Throws: corresponding error
     /// - Important: identity should be the same as in JWT generated at server side
+    /// - Note: To use TouchId or FaceId, in addion `biometricProtection` to being enabled,
+    ///  `NSFaceIDUsageDescription` key should be included in your app’s Info.plist file.
+    /// See [here](https://developer.apple.com/documentation/localauthentication/logging_a_user_into_your_app_with_face_id_or_touch_id)
+    /// more info
+    /// - Note: `biometricalProtection` flag does not resave all entries with new options.
+    /// Use `setBiometricalProtection` method for this purpose
     @objc public convenience init(identity: String,
                                   tokenCallback: @escaping RenewJwtCallback,
+                                  biometricProtection: Bool,
```

Comment:
Too many parameters
1. Add some params struct
2. Add possibility to configure E3Kit using config file

Reply:
1. How would whis help?